pkgs/by-name/wp/wpaperd/package.nix

```diff
@@ -1,35 +1,27 @@
-{ lib, rustPlatform, fetchFromGitHub, pkg-config, libxkbcommon }:
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, libxkbcommon, wayland, libGL }:
 
 rustPlatform.buildRustPackage rec {
   pname = "wpaperd";
-  version = "0.3.0";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
     owner = "danyspin97";
     repo = pname;
```

Comment:
```suggestion
    repo = "wpaperd";
```

Reply:
whats the reasoning for this change? pname being the repo name seems reasonable

Reply:
See https://github.com/NixOS/nixpkgs/issues/277994